bitrise.yml

```diff
@@ -0,0 +1,99 @@
+format_version: 4
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  # If you want to share this step into a StepLib
+  - BITRISE_STEP_ID: mobitru-upload
+  - BITRISE_STEP_VERSION: "0.0.1"
+  - BITRISE_STEP_GIT_CLONE_URL: https://github.com/Mobitru/bitrise-step-mobitru-upload.git
+  - MY_STEPLIB_REPO_FORK_GIT_URL: https://github.com/Mobitru/bitrise-steplib.git
+
+workflows:
+  test:
+    envs:
+    - SAMPLE_IPA_URL: https://github.com/Mobitru/demo-ios-app/releases/download/1.2.0/MobitruUITests-Runner.ipa
```

Comment:
it will be better to use [Mobitru.ipa](https://github.com/Mobitru/demo-ios-app/releases/download/1.2.0/Mobitru.ipa) because the MobitruUITests-Runner.ipa file is just instrumentation tests

Reply:
The test is required **any** valid APK or IPA file. I use this one because of its small size of 3MB.